projects/angular/src/data/datagrid/_datagrid.clarity.scss

```
@@ -2019,148 +1189,75 @@
 
   .datagrid-detail-body {
     flex: 1 1 auto;
-    @include mixins.css-var(
-      padding,
-      clr-datagrid-detail-body-padding,
-      0 variables.$clr_baselineRem_1,
-      variables.$clr-use-custom-properties
-    );
-    @include mixins.css-var(
-      color,
-      clr-datagrid-detail-body-text-color,
-      variables.$clr-p1-color,
-      variables.$clr-use-custom-properties
-    );
+    padding: datagrid-variables.$clr-datagrid-detail-body-padding;
```

`$clr-datagrid-detail-body-padding` is introduced in TM, it's safe to be deleted.
```suggestion
    padding: tokens.$cds-global-space-7 0;
```